mission-lotto/src/main/java/com/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,42 @@
+package com.lotto.domain;
+
+public enum LottoRank {
+
+    FIRST(6, 2000000000),
+    SECOND(5, 1500000),
+    THIRD(4, 50000),
+    FOURTH(3, 5000),
+    NONE(0, 0);
+
+    private final int count;
+    private final int prize;
+
+    LottoRank(int count, int prize) {
+        this.count = count;
+        this.prize = prize;
+    }
+
+    public int getCount() {
+        return count;
+    }
+
+    public int getPrize() {
+        return prize;
+    }
+
+    public static LottoRank getRank(int count) {
+        for (LottoRank rank : values()) {
+            if (rank.getCount() == count) {
+                return rank;
+            }
+        }
+        return NONE;
+    }
+
+    public boolean getRankResult(LottoRank lottoRank) {
+        if (lottoRank == NONE) {
+            return false;
+        } else
+            return true;
+    }
```

Comment:
```suggestion
        if (lottoRank == NONE) {
            return false;
        } else {
            return true;
         }
    }
```

else문이 나쁘다는 건 아니지만 예상치 못한 값이 나올 수 있기 때문에 가급적이면 지양해주세요!

if() { return false}
return true;

이렇게 해줘도 좋을 것 같습니다!